report/src/future_work/future_work.tex

```diff
@@ -0,0 +1,21 @@
+Similarly, the physical layout and the obstacles within a room can affect the classification performed in the second phase of fingerprinting. 
+It is therefore relevant to experiment with several configurations related to beacon placement, number of beacons, physical room layout, and different radio wave noise within the room and in the surrounding environment. 
+
+Future research pursuits within room occupancy detection research, leveraging fingerprinting using BLE, should prioritize the discovery of a configuration that shows improved generalization across diverse settings. 
+One approach to achieving this involves further studies of strategies that can improve either the generated map or the classification algorithm.
+Our experiments showed that the quality of classifications degraded for positions outside the room by up to $49.91$ percent compared to indoor positions, as outlined in Table \ref{lst:resultsPhone_precision}. Therefore, future research should experiment with different beacon configurations where, for example, one or more beacons are placed outside the meeting room. 
+Future experiments might show an improvement in the map quality resulting in better classification. Alternatively, it could show that fewer beacons are needed to achieve similar results. 
+Further to this, an exploration of alternative algorithms in the pursuit of improved generalization would be beneficial.
+Potential candidates might include statistical kNN methods or machine learning approaches like a state vector machine, similar to what \citeauthor{ble_kneares_neural} proposed in their paper \cite{ble_kneares_neural}. 
+Understanding the impacts of algorithm and configuration variations in the experimental setup would be necessary to achieve a more generalized solution.
+
+For Mariendal IT, results found in experimentation, both for the beacon configuration as well as the optimal algorithm, could be integrated into a system inspired by the prototype described in Section \ref{subsec:develop_proto}. 
+In the prototype evaluation, we showed that a system solution for Mariendal IT's meeting room management should gather information about who participates in meetings (see appendix \ref{appendix:user_stories} and \ref{appendix:usability_tasks}).
+A phone application could be implemented for these participants' phones. 
+This application could integrate with the participants' calendar and examine which meeting rooms they attend at specific times.
+This system could use this information to establish which beacons are relevant to report RSSI values from, and send received measurements to the server. 
+The server would then compute a classification of the position of the participants, and use this information to classify the room as occupied or unoccupied. 
+This would allow the meeting participants to end the meeting early by simply leaving the room.
```

Comment:
I'm rather confused by this. We can't copy the beacon configuration, as that is specific to each individual room. So are we just saying that we could integrate kNN into the tablet prototype view?

That isn't really possible either, as it is up to the receiving decide (i.e. the user's phone) to compute their classification.

You could, however, set up communication between the phone and a server - via the app - which the tablet could use to determine whether the meeting is ongoing or not.

But I'm not sure if that's what you're saying. Let me see if I understand clearly:
- Integrate the current system into the prototype (not clear from the text - you write 'results found...')
- Make a phone application (I'd imagine the one we did make would be part of the system as a whole?).
- Integrate the app with the user's calendar - why? Why not just calculate that on the server? You'd only need the app to be a receiver & then send its latest measurement to a server, really. The server could use pre-made data-sets of the meeting rooms on-premise to classify the user's presence.

It seems like the above is trying to get captured, but I can't quite glean it from the writing. Could you try rephrasing with the above points in mind?

LMK if you need clarification.